quickstarts/php/codeigniter/config.yml

```
@@ -16,4 +16,4 @@ keywords:
   - apm
   - php
 installPlans:
-  - php-agent
+  - php-targeted-agent
```

```suggestion
  - php-targeted-install
```
Update this so it matches the `id` of the install plan